beetsplug/missing.py

```diff
@@ -181,8 +229,8 @@ def _missing_albums(self, lib, query):
                 continue
 
             try:
-                resp = musicbrainzngs.browse_release_groups(artist=artist[1])
-                release_groups = resp['release-group-list']
+                resp = musicbrainzngs.browse_releases(artist=artist[1])
+                release_groups = resp['release-list']
```

Comment:
Is this related to the formatting change?

Reply:
It is, but it's in the wrong place, I'll explain after the next commit.